UI/GameInfoCache.cpp

```diff
@@ -375,6 +383,7 @@ class GameInfoWorkItem : public PrioritizedWorkQueueItem {
 						goto handleELF;
 					}
 					ERROR_LOG(LOADER, "invalid pbp %s\n", pbpLoader->Path().c_str());
+					info_->DisposeFileLoader();
```

Comment:
I think I saw a potential leak. But I was mostly trying anything to try to figure out the crashes, which were just that silly mutex thing.